src/components/app/details/triggerView/cdMaterial.tsx

```diff
@@ -507,6 +510,19 @@ export class CDMaterial extends Component<CDMaterialProps, CDMaterialState> {
                         <span className="fs-13 fw-4">{mat.deployedTime}</span>
                     </div>
                 )}
+
+                {(mat.index == 0)&& <div className="mt-6 ml-2">
+                    <ImageTagButton
+                        text={'Latest'}
```

Comment:
it can directly be written as text="Latest"

Reply:
done